dev-cpp/valijson/files/1.0-fix-macro-typos.patch

```diff
@@ -0,0 +1,73 @@
+From f1ff3518281fe347c97b78384d6bb645dbdd4788 Mon Sep 17 00:00:00 2001
```

Comment:
Can you also add the link where this patch originates from?  It is a github commit, but a link is better.